src/controllers/submission.controller.js

```
@@ -5,6 +5,32 @@ import s3 from "../services/storage.js";
 import Submission from "../models/Submission.js";
 import { commentMarkers } from "../config/markers.js";
 
+const getAll = async (req, res) => {
+  const problemId = sanitize(req.query.problemId, "mongo");
+  const language = sanitize(req.query.language, "string");
+  const page = parseInt(req.query.page) || 1;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)